opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -153,9 +153,14 @@ const getUserAndGlobalMarkings = async (context, userId, userGroups, capabilitie
   return { user: computedMarkings, all: markings };
 };
 
-export const getCapabilities = async (context, userId, isUserPlatform) => {
-  const roles = await listThroughGetTo(context, SYSTEM_USER, userId, RELATION_HAS_ROLE, ENTITY_TYPE_ROLE);
-  const roleIds = roles.map((r) => r.id);
+export const getRoles = async (context, userId, userGroups) => {
```

Comment:
`userId` parameter is unused in this method

Reply:
fixed